src/query.ts

```diff
@@ -49,9 +58,19 @@ async function run(): Promise<void> {
         repo.pat
       );
 
+      let queryPack: string | undefined;
+      if (queryPackUrl !== "") {
+        console.log("Getting query pack");
+        const queryPackArchive = await downloadDatabaseFile(
```

Comment:
Not strictly necessary right now but I'm feeling we should rename `downloadDatabaseFile` and the file it's in to something not mentioning databases. It's just a generic method for downloading whatever.